pkg/cmd/roachprod/main.go

```diff
@@ -446,14 +446,12 @@ func setupSSH(clusterName string) error {
 	if err := installCluster.Wait(); err != nil {
 		return err
 	}
-	// If we are using AWS we need to fetch public keys from gcloud to set
-	// up ssh access for other users.
-	if cloudCluster.HasProvider(aws.ProviderName) {
-		installCluster.AuthorizedKeys, err = gce.GetUserAuthorizedKeys()
-		// Don't fail if we can't get other user's authorized keys, just log.
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "failed to retrieve authorized keys from gcloud: %v", err)
-		}
+	// Fetch public keys from gcloud to set up ssh access for all users into the
+	// ubuntu user.
+	installCluster.AuthorizedKeys, err = gce.GetUserAuthorizedKeys()
+	// Don't fail if we can't get other user's authorized keys, just log.
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "failed to retrieve authorized keys from gcloud: %v", err)
```

Comment:
We may actually want to fail here if this is GCE and we move to using the shared user for everything as we won't have access.